pkg/resource/stack/deployment.go

```diff
@@ -478,23 +490,24 @@ func SerializePropertyValue(ctx context.Context, prop resource.PropertyValue, en
 		if showSecrets {
 			secret.Plaintext = plaintext
 		} else {
-			// If the encrypter is a cachingCrypter, call through its encryptSecret method, which will look for a matching
-			// *resource.Secret + plaintext in its cache in order to avoid re-encrypting the value.
-			var ciphertext string
-			if cachingCrypter, ok := enc.(*cachingSecretsManager); ok {
-				ciphertext, err = cachingCrypter.encryptSecret(ctx, prop.SecretValue(), plaintext)
+			// If the encrypter is a EventualSecretEncrypter, call through its EncryptSecret method,
```

Comment:
```suggestion
			// If the encrypter is an EventualSecretEncrypter, call through its EncryptSecret method,
```

Reply:
Also this comment doesn't appear to match the actual type we check for on the next line